### DISCUSSION TITLE
Ironing single layer print (top=bottom)

### DISCUSSION THREAD

**Discussion by DocRuzzy** [supermerill/SuperSlicer#1282] [Q&A]
I believe the Ironniing of "all solid surfaces" is still broken but I actually don't need that. Is the bottom infill option "iron" affected in the same way?  I would like to iron a surface that is a single layer thick (which would be the bottom, though it's also the top). 
This is not currently possible., as a minimum of two layers is required so that one layer is considered the top. (note. this would actually be 3 layers with the ironing layer counting as one layer). I can generate a 2 layer print with ironing on top and manually edit the code to put the ironing on a single layer print and it works (resulting in the bottom layer plus an ironing layer).

Because ironing is not an option for infill patterns in prusaslicer I didn't know if these actions are related.

**Comment by supermerill** [comment]
> I believe the Ironniing of "all solid surfaces" is still broken

? I don't have an issue on that.

> I would like to iron a surface that is a single layer thick (which would be the bottom, though it's also the top).

Be careful, as any spot where the bed is a bit too far away will be very visible.
You can use the pattern "infill" as a bottom pattern. It's exactly the same process as prusaslicer ironing, but it should be good enough.

**Reply by DocRuzzy** [reply]
Thanks for the quick reply, You're right. I've got it working now.
